conf/parser.py

```diff
@@ -147,6 +149,16 @@ def parse(self, ifaces):
             print('Can\'t parse interface name(s)! Setting it to default values ({}, {})'.format(
                 "access", "core"))
 
+        # UnixPort Paths
+        try:
+            self.notify_sockaddr = self.conf["notify_sockaddr"]
+            self.endmarker_sockaddr = self.conf["endmarker_sockaddr"]
+        except KeyError:
+            self.notify_sockaddr = "/tmp/notifycp"
+            self.endmarker_sockaddr = "/tmp/pfcpport"
```

Comment:
you can put this in the init lines above.

Reply:
done